src/main/java/frc/robot/shuffleboard/GRTNetworkTableEntry.java

```diff
@@ -0,0 +1,54 @@
+package frc.robot.shuffleboard;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableValue;
+
+/**
+ * A wrapper for NetworkTableEntry to allow different classes to do the
+ * accessing and the updating. This is so that subsystems can get/set
+ * values from Shuffleboard inexpensively thanks to a separate thread (such as
+ * one created by the ShuffleboardManager) which pushes or pulls the actual
+ * value from Shuffleboard.
+ * 
+ * An alternative to using this class is to use listeners:
+ * https://docs.wpilib.org/en/stable/docs/software/networktables/listening-for-change.html
+ */
+public class GRTNetworkTableEntry {
```

Comment:
Could we use a generic for this? I am unsure if this feature exists in Java 11, but at least from earlier testing for the lambdas gist Java 17 has the capacity to intelligently infer generics, as in:
```java
import java.util.function.Consumer;

public class Main {
    public static void main(String... args) {
        callConsumer(System.out::println, "Hello, world!");
    }

    public static <T> void callConsumer(Consumer<T> consumer, T value) {
        consumer.accept(value);
    }
}
```

Reply:
I know it functions ok for setting, but using `Object` as a catch-all "any" type doesn't necessarily make the most sense to me, especially since actual networktable entries are typed.

Reply:
When we call NetworkTableEntry.setValue(Object), it automatically infers the type. Having stronger protection further up the line might be more correct but we have two options, the first being to construct a NetworkTableValue on GRTNetworkEntry.setValue (which seems redundant) and the second being to force users to pass in a NetworkTableValue to GRTNetworkEntry.setValue(). This'll look like `myGRTNetworkEntry.setValue(new NetworkTableValue(NetworkTableType.kdouble, <thedouble>)` for each and every value